packages/web-components/src/components/masthead/cloud/cloud-masthead-profile.ts

```diff
@@ -0,0 +1,140 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2021
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { ifDefined } from 'lit-html/directives/if-defined';
+import { html, property, query, customElement, LitElement } from 'lit-element';
+import settings from 'carbon-components/es/globals/js/settings';
+import ddsSettings from '@carbon/ibmdotcom-utilities/es/utilities/settings/settings.js';
+import User20 from 'carbon-web-components/es/icons/user/20.js';
+import UserOnline20 from 'carbon-web-components/es/icons/user--online/20.js';
+import FocusMixin from 'carbon-web-components/es/globals/mixins/focus.js';
+import HostListenerMixin from 'carbon-web-components/es/globals/mixins/host-listener.js';
+import HostListener from 'carbon-web-components/es/globals/decorators/host-listener.js';
+import styles from '../masthead.scss';
+
+const { prefix } = settings;
+const { stablePrefix: ddsPrefix } = ddsSettings;
+
+/**
+ * The Cloud-specific profile menu UI in the masthead.
+ *
+ * @element dds-cloud-masthead-profile
+ */
+@customElement(`${ddsPrefix}-cloud-masthead-profile`)
+class DDSCloudMastheadProfile extends HostListenerMixin(FocusMixin(LitElement)) {
```

Comment:
Wondering if this class will become simpler by extending `DDSMastheadProfile`. Seems that most properties/methods are copies.